flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -220,6 +222,8 @@ public void setup() throws IOException {
 		jobManagerLeaderRetriever = new SettableLeaderRetrievalService();
 		haServices.setResourceManagerLeaderRetriever(resourceManagerLeaderRetriever);
 		haServices.setJobMasterLeaderRetriever(jobId, jobManagerLeaderRetriever);
+
+		networkEnvironment = new NetworkEnvironmentBuilder().build();
```

Comment:
should we call `networkEnvironment.start();`? or at least keep it in `testShouldShutDownTaskManagerServicesInPostStop`? Also in `TaskAsyncCallTest`?

Reply:
Actually it is not needed to call start in these tests, because the start would only make `NettyServer` to bind one port to listen and it would not change any internal state. The `NettyServer` is not really needed in these tests. Certainly it would no harm if calling start always. Do you think we should do that in these tests?

Reply:
I only call the start for `testShouldShutDownTaskManagerServicesInPostStop` to keep the same behavior as before. 
For the other tests in `TaskTest` and `TaskAsyncCallTest`, I think it is better to not call start atm for the following concerns:
1. Start would bind port which takes some time for each test
2. The bind port would consume more system resources
3. It is not actually useful to start `NettyServer`.